tests/modules/programs/rofi/wofi-assert-on-plugins.nix

```diff
@@ -0,0 +1,17 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    programs.rofi = {
+      enable = true;
+      package = pkgs.wofi;
+      plugins = [ pkgs.rofi-calc ];
+    };
+
+    test.asserts.assertions.expected = [''
+      Cannot use provided package with plugins.
+    ''];
+  };
```

Comment:
```suggestion
let
  dummyPackage = x: pkgs.runCommandLocal "dummy-${x}" { } "";
in
{
  config = {
    programs.rofi = {
      enable = true;
      package = pkgs.wofi;
      plugins = [ pkgs.rofi-calc ];
    };
    
    nixpkgs.overlays = [
      (self: super: { rofi = dummyPackage "rofi"; rofi-calc = dummyPackage "rofi-calc"; })
    ];

    test.asserts.assertions.expected = [''
      Cannot use provided package with plugins.
    ''];
  };
}
```

Reply:
Shouldn't overlay be 
```nix
    nixpkgs.overlays = [
      (self: super: { wofi = dummyPackage "wofi"; rofi-calc = dummyPackage "rofi-calc"; })
    ];
```
since `config.programs.rofi.package` uses `pkgs.wofi`.

Reply:
Shouldn't overlay be 
```nix
    nixpkgs.overlays = [
      (self: super: { wofi = dummyPackage "wofi"; rofi-calc = dummyPackage "rofi-calc"; })
    ];
```
since `config.programs.rofi.package` uses `pkgs.wofi`.

Reply:
Yes good catch